test/framework/apply.go

```diff
@@ -68,6 +68,15 @@ func KubectlApply(ctx context.Context, kubeconfigPath string, resources []byte,
 	if len(stdout) > 0 {
 		log.Info("Stdout:", "stdout", string(stdout))
 	}
+
+	if err != nil {
+		if strings.Contains(string(stderr), "Error from server (AlreadyExists)") {
```

Comment:
Kubectl apply does not return already exist error. What is the reason for this code block?

Reply:
it does when the same provider is applied twice https://github.com/rancher/turtles/actions/runs/13762403621/job/38481478321#step:9:540

Reply:
```
> kubectl apply -f bundle.yaml 
bundle.fleet.cattle.io/test created
> kubectl apply -f bundle.yaml 
bundle.fleet.cattle.io/test unchanged
```

Reply:
it fails every time in CI when the provider gets applied second time

Reply:
It is probably not related, but UI stopped allowing addition of the same provider twice, the provider icon is disabled if it is already added.